### PR TITLE
made (or at least tried to make) description less confusing

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#1270

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ As you set up or edit your ATP anti-phishing policies, you can choose from sever
...
...tandard way.  <br/> **Aggressive** System handles emails suspected to be phish with a high degree of confidence the same way as those, suspected with very high degree of confidence.  <br/> **More aggressive** System handles emails suspected to be phish with a medium or high degree of confidence the same way as those, suspected with very high degree of confidence.  <br/> **Most aggressive** System handles emails suspected to be phish with a low, medium or high degree of confidence the same way as those, suspected with very high degree of confidence.|When you want to be more aggressive in the treatment of potentially phishing messages within Office 365. For example, messages with a very high probability of being phish will have the most aggressive actions taken on them while messages with a low probability have less aggressive actions taken on them. This setting also impacts other parts of the filtering system that combine signals together. This doesn't necessarily mean different actions are implemented.  Essentially you set the probability of mail being phish, to determine the (same) designated action. The chance of moving good messages increases as the level of settings increases.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md]:
```suggestion
|**Advanced phishing thresholds**|Defines the level of settings for how phishing messages are handled.  <br/> **Standard**: Email suspected to be phish is handled in the standard way.  <br/> **Aggressive**: The system handles emails suspected to be phish with a high degree of confidence, the same way as those suspected with a very high degree of confidence.  <br/> **More aggressive**: The system handles emails suspected to be phish with a medium or high degree of confidence, the same way as those suspected with a very high degree of confidence.  <br/> **Most aggressive**: The system handles emails suspected to be phish with a low, medium, or high degree of confidence, the same way as those suspected with a very high degree of confidence.|When you want to be more aggressive in the treatment of potentially phishing messages within Office 365. For example, messages with a very high probability of being phish will have the most aggressive actions taken on them while messages with a low probability have less aggressive actions taken on them. This setting also impacts other parts of the filtering system that combine signals. This doesn't necessarily mean different actions are implemented.  Essentially you set the probability of mail being phish, to determine the (same) designated action. The chance of moving good messages increases as the level of settings increases.|
```

File: microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ As you set up or edit your ATP anti-phishing policies, you can choose from sever
 |**Enable mailbox intelligence based impersonation protection**|Enables or disables mailbox intelligence for impersonation protection for this policy. The important here is the control of the impersonation for a given mailbox.|When you want to enhance impersonation results for users based on each user's individual sender map. This intelligence allows Office 365 to customize user impersonation detection and better handle false positives. When user impersonation is detected, based on mailbox intelligence, you can define what action to take on the message.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-anti-phishing-policies.md]:
```suggestion
|**Enable mailbox intelligence based impersonation protection**|Enables or disables mailbox intelligence for impersonation protection for this policy. The important aspect here is the control of the impersonation for a given mailbox.|When you want to enhance impersonation results for users based on each user's individual sender map. This intelligence allows Office 365 to customize user impersonation detection and better handle false positives. When user impersonation is detected, based on mailbox intelligence, you can define what action to take on the message.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #1270.

Thank you.

cc: @kenwith

**Review by MSFTTracyP — COMMENTED** [review]
Thanks

**Pull request merged**